<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraSettingsDialog</class>
 <widget class="QDialog" name="CameraSettingsDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camera Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
border: 1px solid rgb(100,100,100);
border-radius: 5px;
margin-top: 2ex;
}

QGroupBox::title {
subcontrol-origin: margin;
left: 10px;
}</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Mouse Sensitivity</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QFormLayout" name="formLayout_6">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Panning</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSlider" name="sldr_pan">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Turning</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSlider" name="sldr_turn">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::NoTicks</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Flying</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Mass</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSlider" name="sldr_mass">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Drag</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSlider" name="sldr_drag">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Thrust</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSlider" name="sldr_thrust">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Panning</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Y Axis</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cmb_pan_invert_y">
          <item>
           <property name="text">
            <string>Not Inverted</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Inverted</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>X Axis</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="cmb_pan_invert_x">
          <item>
           <property name="text">
            <string>Not Inverted</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Inverted</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Camera Center Rotation</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Y Axis</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cmb_center_invert_y">
          <item>
           <property name="text">
            <string>Not Inverted</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Inverted</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>X Axis</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="cmb_center_invert_x">
          <item>
           <property name="text">
            <string>Not Inverted</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Inverted</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Object Center Rotation</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Y Axis</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cmb_object_invert_y">
          <item>
           <property name="text">
            <string>Not Inverted</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Inverted</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>X Axis</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="cmb_object_invert_x">
          <item>
           <property name="text">
            <string>Not Inverted</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Inverted</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cmb_center_invert_y</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>CameraSettingsDialog</receiver>
   <slot>update_params()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmb_center_invert_x</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>CameraSettingsDialog</receiver>
   <slot>update_params()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>408</x>
     <y>382</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmb_object_invert_y</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>CameraSettingsDialog</receiver>
   <slot>update_params()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmb_object_invert_x</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>CameraSettingsDialog</receiver>
   <slot>update_params()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>423</x>
     <y>482</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>368</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sldr_drag</sender>
   <signal>valueChanged(int)</signal>
   <receiver>CameraSettingsDialog</receiver>
   <slot>update_params()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sldr_mass</sender>
   <signal>valueChanged(int)</signal>
   <receiver>CameraSettingsDialog</receiver>
   <slot>update_params()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sldr_thrust</sender>
   <signal>valueChanged(int)</signal>
   <receiver>CameraSettingsDialog</receiver>
   <slot>update_params()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sldr_pan</sender>
   <signal>valueChanged(int)</signal>
   <receiver>CameraSettingsDialog</receiver>
   <slot>update_params()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sldr_turn</sender>
   <signal>valueChanged(int)</signal>
   <receiver>CameraSettingsDialog</receiver>
   <slot>update_params()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmb_pan_invert_y</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>CameraSettingsDialog</receiver>
   <slot>update_params()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmb_pan_invert_x</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>CameraSettingsDialog</receiver>
   <slot>update_params()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>494</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>up()</slot>
  <slot>update_params()</slot>
 </slots>
</ui>
